encrypted-media-respec.html

```diff
@@ -1169,7 +1169,7 @@
         <dd>
           <p class="issue"><a href="https://github.com/w3c/encrypted-media/issues/85">Issue 85</a> - There are concerns about the architectural implications of <code>"persistent-usage-record"</code> sessions that are pending a TAG discussion. The outcome of this discussion could result in modification (including removal) of the feature.</p>
           <p>
-            A session for which the license and any key(s) it contains SHALL NOT be persisted and for which a <a def-id="record-of-key-usage"></a> SHALL be persisted when
+            A session for which the license and key(s) are not persisted and for which a <a def-id="record-of-key-usage"></a> is persisted when
```

Comment:
Is there a reason you switched to non-RFC words? I suppose that reads better as a description, though we do want these to be normatively required behaviors (hopefully also defined in the algorithms). I wonder if these descriptions are supposed to be normative, though.

There are still some RFC words in the following text. In particular, the "SHALL be" for `persistent-license` seems equivalent to the "SHALL BE" that you replaced here. These should be consistent either way.